spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe UsersController do
+  describe "#index" do
```

Comment:
これはキッチハイクのテストコードの特徴なのですが、
`GET /users`というようにHTTPアクションと相対パスを記載してあげてください！
```suggestion
  describe "GET /users" do
```